doc/nrf/known_issues.rst

```diff
@@ -1454,7 +1454,7 @@ In addition to the known issues listed here, see also :ref:`softdevice_controlle
 DRGN-16013: Initiating connections over extended advertising is not supported when external radio coexistence and FEM support are enabled.
   The initiator can assert when initiating a connection to an extended advertiser when both external radio coexistence and FEM are enabled.
 
-  **Workaround:**  Do not enable both coex (:kconfig:`CONFIG_MPSL_CX_BT`) and FEM (:kconfig:`CONFIG_MPSL_FEM`) when support for extended advertising packets is enabled (:kconfig:`CONFIG_BT_EXT_ADV`).
+  **Workaround:**  Do not enable both coex (:kconfig:`CONFIG_MPSL_CX_BT_3WIRE`) and FEM (:kconfig:`CONFIG_MPSL_FEM`) when support for extended advertising packets is enabled (:kconfig:`CONFIG_BT_EXT_ADV`).
```

Comment:
Maybe leave it as-is as I don't think we tried to reproduce the bug when using 1-wire. @Tschet1 any thoughts ?

Reply:
Outdated